FirebaseStorageUI/UIImageView+FirebaseStorage.h

```diff
@@ -0,0 +1,147 @@
+//
+//  Copyright (c) 2016 Google Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+@import UIKit;
+@import SDWebImage;
+@import FirebaseStorage;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol FUIImageCache <NSObject>
+@required
+
+- (nullable UIImage *)imageFromMemoryCacheForKey:(NSString *)key;
+- (nullable UIImage *)imageFromDiskCacheForKey:(NSString *)key;
+- (void)storeImage:(UIImage *)image forKey:(NSString *)key;
+
+@end
+
+@protocol FUIDownloadTask <NSObject>
```

Comment:
I'm fine if we leave them, but they should move to the test class rather than being in the public interface.